cmd/keys.go

```diff
@@ -363,3 +363,44 @@ $ %s k e cosmoshub testkey`, appName, appName)),
 
 	return cmd
 }
+
+// addressCmd represents the address of a relayer
+func addressCmd(a *appState) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "address chain_name [key_name]",
+		Aliases: []string{"a"},
+		Short:   "Shows the address of a relayer",
+		Args:    withUsage(cobra.RangeArgs(1, 2)),
+		Example: strings.TrimSpace(fmt.Sprintf(`
+$ %s address ibc-0
+$ %s address ibc-1 key2
+$ %s a ibc-2 testkey`, appName, appName, appName)),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Rather than duplicate the logic from keysShowCmd, we should pull out the `RunE` into a common method used by both